src/backbone.radio.js

```diff
@@ -168,7 +168,12 @@ _.extend(Radio, {
  */
 
 function makeCallback(callback) {
-  return _.isFunction(callback) ? callback : function () { return callback; };
+  if (_.isFunction(callback)) {
+    return callback;
+  }
+  var result = function () { return callback; };
+  result._callback = callback;
+  return result;
 }
```

Comment:
Want to condense this logic into:

```js
function makeCallback(val) {
  var callback = _.isFunction(val) ? val : function() { return val; };
  callback._original = val;
  return callback;
}
```

Then https://github.com/marionettejs/backbone.radio/blob/master/src/backbone.radio.js#L89 can be simplified to:

```diff
    if (
-      (!callback || (callback === event.callback || callback === event.callback._callback)) &&
+      (!callback || (callback === event.callback._original)) &&
       (!context || (context === event.context))
    ) {
```